### ISSUE TITLE
Bot doesn't work when messaged directly

### ISSUE THREAD

**Issue by samcat116** [cooperpellaton/NachoMemes#35]
When messaging the bot directly, no meme gets created. After sending the bot a valid command, the bot says that it's typing for several seconds, but then sends nothing.

**Comment by cooperpellaton** [comment]
The reason comes from:
```python
guild = str(ctx.message.guild.id)
meme = store.read_meme(guild, template, True)
```
line 104-5 of `bot.py`. 

We presume a DM from a user has the guild set to `None` so when we go to look up the meme in the table we look for a null table. 

Solution: default table. When receiving a message with no guild generate the meme from the default table.

**Comment by cooperpellaton** [comment]
Closed based on hotfix #37.
